main.cpp

```diff
@@ -59,10 +58,31 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
-
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    int dogLength = 4;
+    int dogHeight = 5;
+    int dogBreadth = 6;
+
+    bool isDogMale = true;
+    bool doesDogPoop = true;
+    bool doesDogTalk = false;
+
+    double dogWalkingDistance = 8;
+    double dogWalkingSpeed = 40;
+    double dogWeight = 50;

```

Comment:
Initialize doubles with doubles.  you have written integers on the right side of the `=`
- [ ] revise